### PR TITLE
Reduce Docker image size

### PR DESCRIPTION
**Pull request by ngosang**
Final Docker image size is reduced from 209 MB => 126 MB (uncompressed)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ COPY . .
 RUN mkdir build && \
     cd build && \
     cmake .. \
-        -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+        -DCMAKE_BUILD_TYPE=Release \
```
Comment (KarlStraussberger) [inline Dockerfile:15]: It's smaller, but there is no way to debug a core if the server crashes like in #2305.

We may set up a whole group of images without debugging symbols.

Comment (ngosang) [reply Dockerfile:15]: Official packages in Debian, Ubuntu, Archlinux... don't include debug symbols.
* Is it possible to reduce the size of Debug symbols? I mean, the binary is just 3MB and debug symbols are 100 MB...
* If the previous point is not possible I think it's a good idea to release 2 Docker images. Keep the current tag for slim version and add debug suffix for the fat one eg:`1.9.2-debug`

**Comment by ngosang** [comment]
@KarlStraussberger take a look at `glevel` => https://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html
do you think `-g1` will provide enough information for debugging? Could you make it crash and see what happens?

1. `-DCMAKE_BUILD_TYPE=RelWithDebInfo` (glevel = 2 by default)
gerbera exec => 112 MB
2. `-DCMAKE_BUILD_TYPE=Release && -DCMAKE_CXX_FLAGS=-g1` (glevel = 1)
gerbera exec => 28 MB
3. `-DCMAKE_BUILD_TYPE=Release` (glevel = 0, no debug symbols)
gerbera exec => 4.1 MB

**Comment by KarlStraussberger** [comment]
@ngosang glevel=1 sounds great. Enough to see where it crashed. No intention to debug from the cores.

**Comment by ngosang** [comment]
@KarlStraussberger I think this is ready.
BTW I noticed that there are some packages in the final image that are not required in runtime. The size decreases from 126 MB to 119 MB and Gerbera starts correctly:
* util-linux
* sqlite
* zlib
* fmt
* ffmpeg-libs
* libebml
I can open another PR with this change if you are interested.

**Comment by KarlStraussberger** [comment]
From my perspective they should be needed.

There is a reasonable request to add ffmpeg and even vlc for transcoding.

**Comment by ngosang** [comment]
Ok, you can merge.

**Comment by KarlStraussberger** [comment]
Thanks

**Pull request merged**